.ci/magic-modules/welcome-contributor.sh

```diff
@@ -2,14 +2,20 @@
 
 set -x
 
+ASSIGNEE=$(shuf -n 1 <(printf "danawillow\nrambleraptor\nemilymye\nrileykarson\nSirGitsalot\nslevenick\ntysen"))
```

Comment:
Not crazy about having this list here (we're bound to forget about it), but no use in putting in too much effort to make this perfect.

Reply:
agreed, but don't have a better idea at the moment.